Github_data_to_excel/load_script.sql

```diff
@@ -0,0 +1,35 @@
+
+truncate table  postgres.public.github_staging;
+
+copy postgres.public.github_staging(
+	category,
+	file_names,
+	Timestamp
+)
+FROM 'C:/Users/Public/dataeng_project_output/Github_data_to_excel/output/output.csv'
+DELIMITER ','
+CSV HEADER;
+
+DELETE FROM postgres.public.github_staging where (category, file_names) is null;
+
+DELETE FROM postgres.public.github_target using postgres.public.github_staging where github_staging.category = github_target.category
+and github_staging.file_names = github_target.file_names;
+
```

Comment:
can we also please write sql statement to handle duplicate records in staging table

Reply:
can we please write sql statement to consider latest record